# Request
uint8[] serialized_ct
---
# Response
bool success
